<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameBoard</class>
 <widget class="QDialog" name="GameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="my_timer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="click11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click22">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click25">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click26">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click27">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click28">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click29">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click30">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click10">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click14">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click15">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>480</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click16">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click17">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click18">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click19">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click20">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click21">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click23">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="click24">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius : 25; 
border : 1px solid black;
background-color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ranking:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nickLabel1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nickLabel2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nickLabel3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nickLabel4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Oczekiwanie na pozostałych graczy</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointsLabel4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pointsLabel3">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pointsLabel2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pointsLabel1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
